apps/aecore/src/aecore_app.erl

```diff
@@ -15,11 +15,9 @@
 start(_StartType, _StartArgs) ->
     ok = lager:info("Starting aecore node"),
     ok = application:ensure_started(mnesia),
+    aec_db:load_database(),
```

Comment:
Comment in export of `load_database/0` shall be updated.
Wait for tables timeout shall be either user configurable or infinity.